u3d.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'filesize', '>= 0.1.1' # File sizes prettifier
   spec.add_dependency 'inifile', '>= 3.0.0', '< 4.0.0' # Parses INI files
   spec.add_dependency 'plist', '>= 3.1.0', '< 4.0.0' # Generate the Xcode config plist file
-  spec.add_dependency 'rubyzip', '>= 2.0.0', '< 3.0.0' # Installation of .zip files
+  spec.add_dependency 'rubyzip', '>= 1.0.0', '< 2.0.0' # Installation of .zip files
```

Comment:
can't we just say >= 1.0.0 
why do we need to restrict and prevent those who migrated to 2.0.0?